### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by shub-garg**
Corrected a few language errors.

### CODE REVIEW

**Review by ramyaparimi — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -221,8 +221,8 @@ <h3>Open Source Diversity<br>Berlin</h3>
 				</a>
 				<a class="col-md-4" href="https://diversifyingopensource.github.io" target="_blank" rel="noopener">
 					<img alt="" src="img/projects/diversifyfoss.png" />
-					<h3>Difersify Open Source<br>Bangalore</h3>
-					<p>We help, encourage and celebrate underrepresented people to contribute in open source projects and speak at conferences!</p>
+					<h3>Diversify Open Source<br>Bangalore</h3>
+					<p>We help encourage, and celebrate underrepresented people to contribute to open source projects and speak at conferences!</p>
```
Comment (ramyaparimi) [inline index.html:225]: Help and encourage are two different efforts in this particular context. Please separate them with a comma.

File: index.html [hunk 2]
```diff
@@ -267,7 +267,7 @@ <h3>Outreachy</h3>
 				<a class="col-md-4" href="https://gssoc.tech" target="_blank" rel="noopener">
 					<img alt="" src="img/projects/girlscriptsummerofcode.jpg" />
 					<h3>GirlScript Summer of Code</h3>
-					<p>3 month long Open Source program during summers, with an aim to help beginners get started with Open Source Development while encouraging diversity.</p>
+					<p>3 month long Open Source program during summers, to help beginners get started with Open Source Development while encouraging diversity.</p>
```
Comment (ramyaparimi) [inline index.html:270]: adding 'with an aim' really emphasizes the idea to encourage beginners. May be adding back to the sentence makes it more specific of the goal.

**Review by ramyaparimi — CHANGES_REQUESTED** [review]

**Comment by jonatoni** [comment]
@shub-garg hey :wave: will you be able to update the changes requested by @ramyaparimi please?

**Pull request closed without merging**